runtime/tests/interpreter/attachments_test.go

```diff
@@ -1645,3 +1645,27 @@ func TestInterpretAttachmentResourceReferenceInvalidation(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestInterpretAttachmentsRuntimeType(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("getType()", func(t *testing.T) {
+
+		t.Parallel()
+
+		inter := parseCheckAndInterpret(t, `
+            resource R {}
+            attachment A for R {}
+            fun test() {
+                let r <- create R()
+                let r2 <- attach A() to <-r
+                let a: Type = r2[A]!.getType()
+                destroy r2
+            }
+        `)
+
+		_, err := inter.Invoke("test")
```

Comment:
Maybe return `a` from `test` and assert the returned run-time type's identifier matches the expected attachment type.